xbmc/guilib/Texture.h

```diff
@@ -49,7 +49,6 @@ class CTexture : public CTextureBase
   static std::unique_ptr<CTexture> LoadFromFile(const std::string& texturePath,
                                                 unsigned int idealWidth = 0,
                                                 unsigned int idealHeight = 0,
-                                                bool requirePixels = false,
```

Comment:
The `requirePixels` parameter was originally added in #3704 for RPi platform, but now RPi platform is removed and the parameter doesn't seem to be used anywhere, so it should be okay to remove it.